macros/MDNSidebar.ejs

```diff
@@ -486,46 +486,64 @@ const text = mdn.localStringMap({
 
 <section class="Quick_links" id="Quick_Links">
   <ol>
-    <li><a href="<%=baseURL%>About"><%=text["About_MDN"]%></a>
-      <ol>
-        <li><a href="<%=baseURL%>About"><%=text["About_MDN"]%></a></li>
-        <li><a href="<%=baseURL%>About/Reading"><%=text["MDN_guide_for_readers"]%></a></li>
-        <li><a href="<%=baseURL%>About/Promote"><%=text["Promote_MDN"]%></a></li>
-        <li><a href="<%=baseURL%>Feedback"><%=text["Send_feedback_about_MDN"]%></a></li>
-      </ol>
+    <li class="toggle">
```

Comment:
``{{MDNSidebar}}`` is converted correctly 👍. When we run out of things to do, we can simplify the menus with single items...  Screenshot from ﻿﻿https://developer.mozilla.org/en-US/docs/MDN/Contribute/Howto/Report_a_problem:

<img width="549" alt="pr 789 mdnsidebar" src="https://user-images.githubusercontent.com/286017/45387263-294a4a80-b5db-11e8-81ba-ff7641f47b27.png">